proto/src/determined/api/v1/api.proto

```diff
@@ -442,7 +442,7 @@ service Determined {
       returns (PatchExperimentResponse) {
     option (google.api.http) = {
       patch: "/api/v1/experiments/{experiment.id}"
-      body: "experiment"
+      body: "*"
```

Comment:
in the [patch feature](https://grpc-ecosystem.github.io/grpc-gateway/docs/mapping/patch_feature/) there're 2 ways to do it: hidden and exposed.
it seems we've got an exposed one here, i.e. previously PatchExperiment had payload with experiment at the top level, i.e. `{id: ..., displayName: ....`
and with this change, it'd become nested:
```
{
  experiment: {
    id: ...,
    displayName: ...
  },
  updateMask: ...
}
```

1. This is breaking the current PatchExperiment API
2. I believe the "hidden" approach is nicer for raw REST API users in particular. 

Can you please try the "hidden" way and investigate what needs to be done to support it in typescript bindings?
- I imagine the go side will stay unchanged.
- we'll need to change our python bindings generator to support this, but it's pretty straightforward.